easybuild/easyconfigs/t/Trim_Galore/Trim_Galore-0.6.0-foss-2018b.eb

```diff
@@ -0,0 +1,42 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+easyblock = 'Tarball'
+
+name = 'Trim_Galore'
+version = '0.6.0'
+
+homepage = 'https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/'
+description = """Trim Galore! is a wrapper script to automate quality and 
+adapter trimming as well as quality control, with some added functionality 
+to remove biased methylation positions for RRBS sequence files (for directional, 
+non-directional (or paired-end) sequencing)."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/FelixKrueger/TrimGalore/archive/']
+sources = [{
+    'filename': '39e1ad7.tar.gz',
+}]
+
+checksums = [
+    'aad008ff2e9605079655013bd91615ca2272048b151b2fe7ee197d18b1b87908',  # 39e1ad7.tar.gz
+    'd4d812854755daf91c2443d268afabdacbd1f0820891aaeafb6ddd056571b3a3',  # perl.patch
+]
+
+dependencies = [
+    ('FastQC', '0.11.8', '-Java-1.8', True),
+    ('cutadapt', '1.18', '-Python-2.7.15')
```

Comment:
@sassy-crick Since `Python` is a runtime dependency, this should trickle up via `versionsuffix`, at which point we basically get the easyconfig already merged via #7893, so closing this.

In addition, the `sources` spec shouldn't be using a commit ID...